VirtoCommerce.Domain/Catalog/Model/Category.cs

```diff
@@ -48,7 +48,38 @@ public Category()
         #endregion
 
         #region IHasOutlines members
-        public ICollection<Outline> Outlines { get; set; } 
+        public ICollection<Outline> Outlines { get; set; }
         #endregion
+
+        public virtual void ReduceDetails(string responseGroup)
+        {
+            //Reduce details according to response group
+            var categoryResponseGroup = EnumUtility.SafeParseFlags(responseGroup, CategoryResponseGroup.Full);
+
+            if (!categoryResponseGroup.HasFlag(CategoryResponseGroup.WithImages))
+            {
+                Images = null;
+            }
+
+            if (!categoryResponseGroup.HasFlag(CategoryResponseGroup.WithLinks))
+            {
+                Links = null;
+            }
+
+            if (!categoryResponseGroup.HasFlag(CategoryResponseGroup.WithProperties))
+            {
+                Properties = null;
```

Comment:
Maybe we could clear PropertyValues (need to verify if they are cleared in platform-core).

Reply:
it's ok with PropertyValues. 
Added missed field "Parents "